tests/roles/common_defaults/defaults/main.yaml

```diff
@@ -1,13 +1,153 @@
+
+# Header for the source databases initial clusters members health check
+mariadb_members_env: |-
+  {{ shell_header }}
+  {{ cells_env }}
+
+  {% for cell in cells %}
+  declare -A SOURCE_GALERA_MEMBERS_{{ cell.upper() }}
+  SOURCE_GALERA_MEMBERS_{{ cell.upper() }}=(
+    {% for n in source_galera_members[cell] | default([]) %}
+    ["{{ n.name }}"]={{ n.ip }}
+    {% endfor %}
+  )
+  {% endfor %}
+
+# Headers for the destination databases access passwords
+mariadb_passwords_env: |-
+  {{ shell_header }}
+  {{ cells_env }}
+
+  declare -A PODIFIED_DB_ROOT_PASSWORD
+  for CELL in $(echo "super $RENAMED_CELLS"); do
+    PODIFIED_DB_ROOT_PASSWORD[$CELL]=$(oc get -o json secret/osp-secret | jq -r .data.DbRootPassword | base64 -d)
+  done
+
+# Header for the source database access
+mariadb_copy_shell_vars_src: |-
+  {{ shell_header }}
+  {{ mariadb_image_env }}
+  {{ cells_env }}
+  MARIADB_CLIENT_ANNOTATIONS='--annotations=k8s.v1.cni.cncf.io/networks=internalapi'
+  STORAGE_CLASS={{ storage_class_name }}
+
+  declare -A TRIPLEO_PASSWORDS
+  for CELL in $(echo $CELLS); do
+    if [ "$CELL" = "default" ]; then
+      TRIPLEO_PASSWORDS[default]="$HOME/overcloud-passwords.yaml"
+    else
+      # in a split-stack source cloud, it should take a stack-specific passwords file instead
+      TRIPLEO_PASSWORDS[$CELL]="$HOME/overcloud-passwords.yaml"
+    fi
+  done
+
+  declare -A SOURCE_DB_ROOT_PASSWORD
+  for CELL in $(echo $CELLS); do
+    SOURCE_DB_ROOT_PASSWORD[$CELL]=$(cat ${TRIPLEO_PASSWORDS[$CELL]} | grep ' MysqlRootPassword:' | awk -F ': ' '{ print $2; }')
+  done
+
+  declare -A SOURCE_MARIADB_IP
+  {% for cell in cells %}
+  SOURCE_MARIADB_IP[{{ cell }}]={{ source_mariadb_ip[cell] }}
+  {% endfor %}
+
+# Header for the destination database access
+mariadb_copy_shell_vars_dst: |
+  {{ shell_header }}
+  {{ oc_header }}
+  {{ mariadb_image_env }}
+  {{ cells_env }}
+
+  # The CHARACTER_SET and collation should match the source DB
+  # if the do not then it will break foreign key relationships
+  # for any tables that are created in the future as part of db sync
+  CHARACTER_SET=utf8
+  COLLATION=utf8_general_ci
+
+  {{ mariadb_passwords_env }}
+
+  declare -A PODIFIED_MARIADB_IP
+  for CELL in $(echo "super $RENAMED_CELLS"); do
+    if [ "$CELL" = "super" ]; then
+      PODIFIED_MARIADB_IP[$CELL]=$(oc get svc --selector "mariadb/name=openstack" -ojsonpath='{.items[0].spec.clusterIP}')
+    else
+      PODIFIED_MARIADB_IP[$CELL]=$(oc get svc --selector "mariadb/name=openstack-$CELL" -ojsonpath='{.items[0].spec.clusterIP}')
+    fi
+  done
+
+# Header for the destination cloud EDPM Nova cell computes FDQN and IP pairs, per a cell
+edpm_computes_shell_vars_src: |-
+  {{ shell_header }}
+  {{ cells_env }}
+
+  {% for cell in renamed_cells %}
+  declare -A COMPUTES_{{ cell.upper() }}
+  COMPUTES_{{ cell.upper() }}=(
+    {% for _, v in edpm_nodes[cell] | default({}) %}
+    ["{{ v.hostName }}"]={{ v.ansible.ansibleHost }}
```

Comment:
I'm stuck here with ``Error was a <class 'ValueError'>, original message: too many values to unpack (expected 2)"}``

The edpm_nodes data that I am using is:
```
edpm_nodes:
  cell1:
    edpm-compute-2:
      hostName: edpm-compute-2.localdomain
      ansible:
        ansibleHost: 192.168.122.102
      networks:
        - defaultRoute: true
          fixedIP: 192.168.122.102
          name: ctlplane
          subnetName: subnet1
        - name: internalapi
          subnetName: subnet1
        - name: storage
          subnetName: subnet1
        - name: tenant
          subnetName: subnet1
        - name: storagemgmt
          subnetName: subnet1
  cell2:
    edpm-compute-4:
      hostName: edpm-compute-4.localdomain
      ansible:
        ansibleHost: 192.168.122.104
      networks:
        - defaultRoute: true
          fixedIP: 192.168.122.104
          name: ctlplane
          subnetName: subnet1
        - name: internalapi
          subnetName: subnet1
        - name: storage
          subnetName: subnet1
        - name: tenant
          subnetName: subnet1
        - name: storagemgmt
          subnetName: subnet1
```